installer/manifests/keptn/templates/remediation-service.yaml

```diff
@@ -18,6 +18,10 @@ spec:
         app.kubernetes.io/name: remediation-service
     spec:
       {{- include "keptn.common.pod-security-context" . | nindent 6 }}
+      initContainers:
+        {{- include "keptn.initContainers.wait-for-nats" . | nindent 8 }}
+        {{- include "keptn.initContainers.wait-for-mongodb-datastore" . | nindent 8 }}
+        {{- include "keptn.initContainers.wait-for-shipyard-controller" . | nindent 8 }}
```

Comment:
I think that this service uses the API of the resource-service

Reply:
added the resource service